app/routines/update_user_salesforce_info.rb

```diff
@@ -10,34 +10,52 @@ def self.call(allow_error_email: false)
   end
 
   def call
-    return if !SalesforceUser.any? && !is_real_production?
+    return if !OpenStax::Salesforce.ready_for_api_usage?
 
```

Comment:
yeah this is a weird thing with my brain.  for some reason I read `if !blah` just fine but every time I see `unless blah` I have to think about it for too long. :-)